nixos/modules/i18n/input-method/fcitx5.nix

```diff
@@ -29,5 +29,7 @@ in
         QT_IM_MODULE = "fcitx";
         XMODIFIERS = "@im=fcitx";
       };
+
+      services.xserver.displayManager.sessionCommands = "${fcitx5Package}/bin/fcitx5 -d";
```

Comment:
Shouldn't this be rather a systemd service?

Reply:
Right, I repace it with a user systemd service.